crates/next-custom-transforms/tests/errors.rs

```diff
@@ -89,45 +89,20 @@ fn next_ssg_errors(input: PathBuf) {
     );
 }
 
-#[fixture("tests/errors/react-server-components/server-graph/**/input.js")]
-fn react_server_components_server_graph_errors(input: PathBuf) {
-    use next_custom_transforms::transforms::react_server_components::{Config, Options};
-    let output = input.parent().unwrap().join("output.js");
-    test_fixture(
-        syntax(),
-        &|tr| {
-            server_components(
-                FileName::Real(PathBuf::from("/some-project/src/layout.js")).into(),
-                Config::WithOptions(Options {
-                    is_react_server_layer: true,
-                    dynamic_io_enabled: false,
-                }),
-                tr.comments.as_ref().clone(),
-                None,
-            )
-        },
-        &input,
-        &output,
-        FixtureTestConfig {
-            allow_error: true,
-            module: Some(true),
-            ..Default::default()
-        },
-    );
-}
-
-#[fixture("tests/errors/react-server-components/client-graph/**/input.js")]
-fn react_server_components_client_graph_errors(input: PathBuf) {
+#[fixture("tests/errors/react-server-components/**/input.js")]
+fn react_server_components_errors(input: PathBuf) {
```

Comment:
Fly-by refactor to combine the `server-graph` and `client-graph` tests. Their config is identical apart from `is_react_server_layer`, which we now derive from the input path, i.e. by checking whether `server-graph` is included in the path.